packages/monitors/README.md

```diff
@@ -0,0 +1,31 @@
+# @uma/monitors
+
+This package contains four UMA monitor bot implementations: `BalanceMonitor`, `ContractMonitor`, `CRMonitor`, and `SyntheticPegMonitor`. The monitor bots are used to monitor the UMA ecosystem for key events. They have a number of monitor modules built into them that enable real time reporting on key events for a given EMP contract.
+
+For more information about running a monitor bot, see the [docs](https://docs.umaproject.org/developers/bots).
+
+## Installing the package
+
+```bash
+yarn add @uma/monitors
+```
+
+## Running the monitors
+
+The config below will start up a monitor bot that will: (1) send messages when new liquidations, alerts, or disputes occur and (2) fire if there is large volatility in the synthetic or price of the underlying. It won't report on any wallet or CR monitoring as no params have been defined.
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=your.node.url MNEMONIC="your mnemonic here" yarn monitors --network mainnet_mnemonic
+```
+
+## Monitors
+
+The four monitors available are:
+
+1. The `BalanceMonitor` takes a specified list of addresses to monitor and sends alerts if their collateral, synthetic or Ether balance drops below defined thresholds.
```

Comment:
nit: the data structures for this is a bit complex. we should perhaps show an example config that contains all the execution examples. WDYT?

Something like this:
```
 "walletsToMonitor": [
          {
            "name": "Monitored sponsor",
            "address": "0x...",
            "crAlert": 1.5
          }
        ],
```

The same goes for all other sections and monitors. it can be a bit of a pain to configure them so examples to copy-paste would be useful. You can make a redacted version of the `bot-configs` repo?